src/rand.rs

```diff
@@ -39,7 +39,7 @@ pub fn timestamp_cycles() -> u64 {
 }
 
 /// Generate pseudo random u32 numbers based on the current timestamp.
-pub fn xor_psuedo_rng_u32() -> u32 {
+pub fn xor_pseudo_rng_u32() -> u32 {
```

Comment:
Can you add this in a separate commit and specify that it fixes a typo in the function name? This will be a breaking change that we need to specify in the CHANGELOG, and otherwise it's easy to miss them.

Reply:
Sure, I'll add it in a separate commit in the same PR.

Reply:
Thanks! Can you please give more details in your commit message? Specify which which functions you renamed to what.

Reply:
The public function is mandatory because we need to include it in the changelog. You can also specify the other ones.